lib/appcat-crossplane.libsonnet

```diff
@@ -77,4 +77,13 @@ local sync_options = {
   */
   CompositeResourceDefinition(name):
     kube._Object(api_version.apiextensions, 'CompositeResourceDefinition', name) + sync_options,
+
+  /**
+  * \brief Helper to create DeploymentRuntimeConfig objects.
+  *
+  * \arg The name of the DeploymentRuntimeConfig.
+  * \return A DeploymentRuntimeConfig object.
+  */
+  DeploymentRuntimeConfig(name):
```

Comment:
This is unused?

Also there's already a helper in common, which is actively used: https://github.com/vshn/component-appcat/blob/master/component/common.libsonnet#L177

Reply:
removed